srv/modules/utils/cluster.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=modernize-parse-error
+#
+# The salt-api calls functions with keywords that are not needed
+# pylint: disable=unused-argument
+"""
+Functions related to the ceph cluster
+"""
+
+
+from __future__ import absolute_import
+from __future__ import print_function
+import logging
+
+
+log = logging.getLogger(__name__)
+
+
+def name():
+    """
+    Return the cluster name from the ceph namespace, original namespace
+    or default to 'ceph'
+    """
+    if ('ceph' in __pillar__ and 'cluster' in __pillar__['ceph']):
+        return __pillar__['ceph']['cluster']
+    elif 'cluster' in __pillar__:
+        return __pillar__['cluster']
+    else:
```

Comment:
The only issue here is that the proposal runner is driven by the admin rather than the pillar.  The intent is that the cluster name could be set, maybe via parameter.  We can investigate later, but hardcoding this to returning 'ceph' now is okay.